src/main/java/Alex.java

```diff
@@ -0,0 +1,79 @@
+import java.util.Scanner;
+
+public class Alex {
+    public static void main(String[] args) {
+        Task[] allTasks = new Task[100];
```

Comment:
I don't like the usage of a magic literal here, perhaps you can extract it to a variable which better explains what the 100 means here?